library/waku_thread/waku_thread.nim

```diff
@@ -95,12 +95,16 @@ proc createWakuThread*(): Result[ptr Context, string] =
 
   return ok(ctx)
 
-proc stopWakuNodeThread*(ctx: ptr Context) =
+proc stopWakuNodeThread*(ctx: ptr Context): Result[void, string] =
```

Comment:
Out of the scope of this PR but I just realised it worth keeping consistency with the "start" proc:
```suggestion
proc stopWakuThread*(ctx: ptr Context): Result[void, string] =
```